app/src/main/java/com/microsoft/device/samples/dualscreenexperience/presentation/MainActivity.kt

```diff
@@ -189,6 +190,7 @@ class MainActivity : AppCompatActivity() {
         if (windowLayoutInfo.isInDualMode() != layoutInfoViewModel.isDualMode.value) {
             tutorial.hide()
             invalidateOptionsMenu()
+            Log.i("HISTORY_TEST", "spanned")
```

Comment:
nit, Any reason about using more `i` than `d` for debug purposes? just curious :-)

Reply:
ooh good catch, no specific reason! I will actually move these log statements to another branch, they were mainly so Bianca and I could investigate some navigational bugs 😄

Reply:
got it! thanks for sharing!